changelogs/fragments/openssl_pkcs12_content.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - openssl_pkcs12 - The module now supports ``certificate_content``/``other_certificates_content`` to match ``privatekey_content`` (https://github.com/ansible-collections/community.crypto/pull/848).
```

I would replace "to match `privatekey_content`" with something that explains what you use them for. `privatekey_content` is there to avoid having to write private keys to disk. You don't need to avoid writing certificates to disk for security reasons.
```suggestion
  - openssl_pkcs12 - the module now supports ``certificate_content``/``other_certificates_content`` to match ``privatekey_content`` (https://github.com/ansible-collections/community.crypto/issues/847, https://github.com/ansible-collections/community.crypto/pull/848).
```